### PR TITLE
Resolve failing codeclimate reporting

### LINKED ISSUE #1531: Travis CI from forked repositories
**Describe the bug**
When following guidelines set in [CONTRIBUTING.md](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md#contributing), I get the following error from Travis:
`The command "xvfb-run -a npm run test:prod:travis" exited with 1.`
_https://travis-ci.org/rhell/simorgh/jobs/514624945_

The build does also not report to the [original GitHub PR](https://github.com/bbc/simorgh/pull/1498).

By **not** changing the `README.md` as instructed to do so in the Contributing guidelines, that above error does not appear, and the [Travis CI builds pass](https://travis-ci.org/rhell/simorgh/builds/514627800). However, this is still not reported on the GitHub PR.

**To Reproduce**

1. Fork the Simorgh repository
2. Follow instructions set out in https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md#contributing
3. Open a PR
4. Visit https://travis-ci.org and watch your build
4. See build fail, and not report into the Github PR

**Expected behaviour**
When committing to a PR on a forked repository, Travis should not error, and should also report the result back to the original GitHub PR.

https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md#run-tests-on-your-fork should be updated.

**Screenshots**
image

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by alex-magana**
Resolves #1531

Depends on https://github.com/bbc/simorgh/pull/1658

**Overall change:** This PR adds the `CC_TEST_REPORTER_ID` in `.travis.yml` to resolve the
`The command "xvfb-run -a npm run test:prod:travis" exited with 1.` issue in builds run on a fork of simorgh.

**Code changes:**

- Add `CC_TEST_REPORTER_ID` to `.travis.yml`. This is required to enable branches of simorgh forks to report test coverage.
- Add a note in `CONTRIBUTING.md` to indicate the preset `CC_TEST_REPORTER_ID` in the `.travis.yml`. With this, contributors shouldn't need to obtain the `CC_TEST_REPORTER_ID` from their respective accounts to enable test reporting.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by alex-magana** [comment]
The `CC_TEST_REPORTER_ID ` set in https://travis-ci.org/bbc/simorgh/settings
needs to be removed, as part of the merging of this PR.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 language: node_js
+env:
+  - CC_TEST_REPORTER_ID=f5b40344b76ccf66fbab5aa40bd5db671464c74f255cc3d4063f8bcd7c0812cf
```
Comment (DemonusPC) [inline .travis.yml:3]: Can this ID be used as an authentication key? What does this ID represent?

Comment (alex-magana) [reply .travis.yml:3]: The ID is a repository identifier used by the CodeClimate test reporter to associate the reported coverage with the correct repository on CodeClimate.
As per, https://docs.codeclimate.com/docs/finding-your-test-coverage-token#section-should-i-keep-my-test-reporter-id-secret, the ID can't be used to access any data.

Comment (DemonusPC) [reply .travis.yml:3]: alright. sound good to me

**Review by DemonusPC — APPROVED** [review]
dank

**Review by chris-hinds — APPROVED** [review]
Need to update this branch with the base branch, but it looks good to me.

**Comment by alex-magana** [comment]
Jenkins trigger trial.

**Comment by alex-magana** [comment]
Reopen to test Jenkins trigger.

**Review by jamesbhobbs — APPROVED** [review]

**Review by 12 — APPROVED** [review]
LGTM

**Comment by alex-magana** [comment]
The branch in the fork repo from which the PR was raised has since been deleted..
Closing this in favour of https://github.com/bbc/simorgh/pull/1781

**Comment by alex-magana** [comment]
The branch in the fork repo from which the PR was raised has since been deleted.
Closing this in favour of https://github.com/bbc/simorgh/pull/1781

**Pull request closed without merging**